tests/unit/test_install.py

```diff
@@ -1002,3 +1002,232 @@ def test_uninstall_no_config_file(ws, mocker):
     ws.workspace.download = lambda _: io.BytesIO(config_bytes)
     ws.workspace.get_status.side_effect = NotFound(...)
     install.uninstall()
+
+
+def test_save_config_with_prefix(ws):
```

Comment:
The logic is in the ConfigureGroups class, move tests there. It doesn't belong to installer, which is one level above configure groups logic

WorkspaceInstaller does too much already and it's difficult to support unit tests that create a config file.

Reply:
Done